package.json

```diff
@@ -5,6 +5,7 @@
 	"license": "CC BY-NC 4.0",
 	"homepage": "https://cdds.component-driven.io",
 	"scripts": {
+		"prevalidate": "npm run generate",
```

Comment:
This is very dangerous — it will delete all participant's work every time they restart the server.

Reply:
Yeah I didn’t think of that. Thank you!